doc/nrf/ug_wifi.rst

```diff
@@ -0,0 +1,141 @@
+.. _ug_wifi:
+
+Wi-Fi
+#####
+
+.. contents::
+   :local:
+   :depth: 2
+
+Wi-Fi is the branding used by the Wi-Fi Alliance to describe the family of wireless local area network standards specified by the IEEE802.11 Working Group.
```

Comment:
Hyperlinks to Wi-Fi Alliance and IEEE802.11 WG.
https://www.wi-fi.org/
https://www.ieee802.org/11/